tools/generate-icons.js

```diff
@@ -8,11 +8,20 @@ import xml2js from 'xml2js';
 const inputSVGFolder = path.resolve('public/img');
 const outputReactIconFolder = path.resolve('src/js/icons');
 
+// Figma converts colors to their color names. Convert them back to hex codes.
+// We do this so the rules in StyledIcon work. '#' indicates a fill or stroke
+// is desired. Otherwise, it is treated like 'none'.
+function convertColors(text) {
```

Comment:
What about colors such as `red` or `blue`? should we be adding it as a need arise?

Reply:
Add as the need arises. There are a lot of named colors. I'm not sure how Figma is behaving for them. Black and white are all we need at the moment.